Feature branch

rehanullah748 opened the pull request.

Nemwel-Boniface commented on index.html:
```diff
@@ -0,0 +1,39 @@
+    <link rel="stylesheet" href="style.css">
+</head>
+<body>
+    <div class="top-bar">
```
- [ ] Great job setting up your portfolio header :clap:. However, it would be semantically correct to have this section wrapped inside a `<header>` tag instead of a `<div>`. Kindly fix this. For a visual of what I mean, below I have included the HTML:
```
    <header class="top-bar">
        <a href="#" class="logo">My logo</a>
        <nav class="menu"><a class="menu-icon" href=""><img src="images/menu.png" alt=""></a></nav>
    </header>
```

Nemwel-Boniface commented on index.html:
```diff
@@ -0,0 +1,39 @@
+</head>
+<body>
+    <div class="top-bar">
+        <div class="logo">My logo</div>
```
- [ ] It would be a UI/ UX boost to your project if you'd make this to be a link instead of a plain `<div>`. This is mostly because users usually click on the logo to redirect them to the main/ landing section of the page. For a visual of what I mean, below is an example:

```
<a href="#" class="logo">My logo</a>
```

Nemwel-Boniface commented on index.html:
```diff
@@ -0,0 +1,39 @@
+<body>
+    <div class="top-bar">
+        <div class="logo">My logo</div>
+        <div class="menu"><a class="menu-icon" href=""><img src="images/menu.png" alt=""></a></div>
```
- [ ] It would be semantically correct to wrap your navigation bars inside a `<nav>` instead of a `<div>` as you have provided. For a visual, here is an example:

```
 <nav class="menu"><a class="menu-icon" href=""><img src="images/menu.png" alt=""></a></nav>
```

Nemwel-Boniface commented on index.html:
```diff
@@ -0,0 +1,39 @@
+    </div>
+    <div class="circle-bottom"><img src="images/Ellipse15.png" alt=""></div>
+        <div class="ellipse-bottom-center"><img src="images/Ellipse4.png" alt=""></div>
+        <div class="ellipse-bottom-right"><img src="images/Ellipse12.png" alt=""></div>
```
- [ ] For semantic reasons, this whole section should be wrapped inside a `<section>` tag as it is a stand-alone section of your project. Kindly fix this. For a visual of what I mean, view the code snippet below:

```
<section>
<div class="image-icons">
        <div class="left-ellipse"><img src="images/Ellipse11.png" alt=""></div>
        <div class="center-ellipse1"><img src="images/Ellipse5.png" alt=""></div>
        <div class="center-rectangle"><img class="center-rect" src="images/Rectangle7.png" alt=""></div>
        <div class="center-ellipse2"><img src="images/Ellipse8.png" alt=""></div>
        <div class="right-ellipse"><img src="images/Ellipse3.png" alt=""></div>
        <div class="right-rectangle"><img src="images/Rectangle4.png" alt=""></div>
    </div>
    <div class="text"><h1>I'm Rehan <br>glade to see <br>you!</h1>
    <p class="paragraph">I’m a software developer! I can help you <br> build a product , feature or website Look <br> through some of my work and <br> experience! If you like what you see and <br>  have a project you need coded, don’t <br> hestiate to contact me.</p>
    </div>
    <div class="social-contact">
        <h1 class="header">LET'S CONNECT</h1>
        <img class="social-icons" src="images/twitter.png" alt="">
        <img class="social-icons" src="images/medium.png" alt="">
        <img class="social-icons" src="images/github.png" alt="">
        <img class="social-icons" src="images/angelist.png" alt="">
        <img class="social-icons" src="images/angellist.png" alt="">
    </div>
    <div class="circle-bottom"><img src="images/Ellipse15.png" alt=""></div>
        <div class="ellipse-bottom-center"><img src="images/Ellipse4.png" alt=""></div>
        <div class="ellipse-bottom-right"><img src="images/Ellipse12.png" alt=""></div>
</section>
```

Nemwel-Boniface commented on index.html:
```diff
@@ -0,0 +1,39 @@
+    <p class="paragraph">I’m a software developer! I can help you <br> build a product , feature or website Look <br> through some of my work and <br> experience! If you like what you see and <br>  have a project you need coded, don’t <br> hestiate to contact me.</p>
+    </div>
+    <div class="social-contact">
+        <h1 class="header">LET'S CONNECT</h1>
```
- [ ] KIndly note that it is HTML best practice to only have one `<h1>` tag for each page. Kindly change this `<h1>` to any other header eg h2-h5.

Nemwel-Boniface commented on index.html:
```diff
@@ -0,0 +1,39 @@
+        <div class="center-rectangle"><img class="center-rect" src="images/Rectangle7.png" alt=""></div>
+        <div class="center-ellipse2"><img src="images/Ellipse8.png" alt=""></div>
+        <div class="right-ellipse"><img src="images/Ellipse3.png" alt=""></div>
+        <div class="right-rectangle"><img src="images/Rectangle4.png" alt=""></div>
```
- [ ] You have done an amazing job aligning your social media icons :clap: . However, kindly note that it would make much more sense if they were actually links which when clicked will re-direct your user to your social media page. 
- [ ] It would also be semantically correct to have them inside an list, either ordered or un-orderd. For an example of what I mean, kindly have a look at the code snippet below:

```
<ul class="image-icons">
   <li class="left-ellipse><a href="#"><img src="images/Ellipse11.png" alt=""></a></li>
</ul>

Nemwel-Boniface reviewed (changes requested):
## Hello @rehanullah748     :wave: , 
I'm Nemwel your reviewer for this PR :man_beard: 

Good job so far in setting your portfolio mobile version. I commend you because I really like the design! :clap: 
There are some issues that you still need to work on to go to the next project but you are almost there!

![](https://media.giphy.com/media/KyBesf8kNT6s88SEgb/giphy.gif)

## Highlights

Great job setting up the correct linters ✔️ 
Professional documentation ✔️ 
Correct workflow ✔️ 
Meaningful commit messages ✔️


## ♻️ Required Changes ♻️

- [ ] I have realized that your project has a horizontal scroll which is not supposed to be there. Kindly note that this adversely affects your portfolio landing page design which is currently looking very good. Kindly fix this. You can start off with the following and build on top of it

```
*{
  margin: 0;
  padding: 0;
  box-sizing: border-box;
}

html, body {
  overflow-x: hidden;
}
```
- [ ] Kindly make sure that your design closely resembled the chosen [Figma design](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=39%3A122). Currently the heading section padding and margin do not match the chosen Figma design. Kindly fix this.

**This is what your design currently looks like:**
![image](https://user-images.githubusercontent.com/86318284/178052058-42a4d70e-3c8e-4c9c-9c8f-7eca0159d0f2.png)



**This is the desired Figma design:**
![image](https://user-images.githubusercontent.com/86318284/178052249-a98006c6-7a36-479b-8c29-5fe58f2c2320.png)




_Check the comments under the review._

## [Optional] suggestions

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding! :partying_face: 👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me **@Nemwel-Boniface** in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

wuletawwonte commented:
Hi <STUDENT_MENTION>,

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

You have submitted a project, that fails linter checks and with any empty pull request description.

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready :pray:

Cheers and Happy coding!:clap::clap::clap:

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

Reem-lab reviewed (commented):
Hi <STUDENT_MENTION>, 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

You have submitted a project, that has conflicts that should be solved please solve it the send the PR again 🔥🔥🙏🏻🙏🏻

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

prolajumokeoni reviewed (approved):
Hi @rehanullah748, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 
## Highlights
- No linter errors ✅ 
- Git flow was used correctly 🚀 


## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- [ ] Adding a  descriptive pull request message, it should be a brief summary describing changes introduced in your pr.


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification @prolajumokeoni.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

rehanullah748 committed:
.gitignore file created

.gitignore file created

rehanullah748 committed:
.gitignore file updated

.gitignore file updated

rehanullah748 committed:
README file updated

README file updated

rehanullah748 committed:
linters.yml file created

linters.yml file created

rehanullah748 committed:
linters.yml file updated

linters.yml file updated

rehanullah748 committed:
index.html file created

index.html file created

rehanullah748 committed:
style.css file created

style.css file created

rehanullah748 committed:
index.html file updated

index.html file updated

rehanullah748 committed:
style.css file updated

style.css file updated

rehanullah748 committed:
style.css file updated

style.css file updated

rehanullah748 committed:
style.css updated

style.css updated

rehanullah748 committed:
style.css file updated

style.css file updated

rehanullah748 committed:
style.css file updated

style.css file updated

rehanullah748 committed:
style.css file updated

style.css file updated

Abenezer-Tilahun commented on index.html:
```diff
@@ -0,0 +1,44 @@
+    <link rel="stylesheet" href="style.css">
+</head>
+<body>
+    <header class="top-bar">
+        <a href="#" class="logo">My logo</a>
+        <nav class="menu"><a class="menu-icon" href=""><img src="images/menu.png" alt=""></a></nav> 
+        </header>
```
- [ ] Good job for using `header` tag for toolbar section , `a` for logo and `nav` tag for hamburger menu. But for hamburger menu you try to import `menu.png` form images folder unfortunately there is no images folder in your Repo, so please try to fix it
![review](https://user-images.githubusercontent.com/93598189/185222152-5032f6b4-259d-4a0d-aaa8-894f422a4d6f.PNG)
.

Abenezer-Tilahun commented on index.html:
```diff
@@ -0,0 +1,44 @@
+            <li><div class="right-ellipse"><img src="images/Ellipse3.png" alt=""></div></li>
+            <li><div class="right-rectangle"><img src="images/Rectangle4.png" alt=""></div></li>
+        </ul>
+        </div>
```
- [ ] For background image you try to import images from images folder but unfortunately there is no images folder in your Repo, so please try to fix it.

Abenezer-Tilahun commented on index.html:
```diff
@@ -0,0 +1,44 @@
+    
+</body>
+</html>
+
```
- Please try to  align the elements according to the requirement, you should use Flexbox in all 2 sections(toolbar and headline section.)

- Both toolbar and headline section must have the margins (top, bottom, left, right) defined in the Template design.
-    You must stick to the design as much as possible (e.g., font, colors, images, text, margins) using [the templates in Figma](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=0%3A1)

Your project looks like this
![Backgr](https://user-images.githubusercontent.com/93598189/185227809-6791d511-dfd2-4dc3-abb8-899fa7d39825.PNG)

The Figma design looks like :

![Figma design](https://user-images.githubusercontent.com/93598189/185227877-12c9398c-c2e2-4a87-82ed-bb34053f2750.PNG)

Abenezer-Tilahun commented on style.css:
```diff
@@ -0,0 +1,149 @@
+.social-contact {
+  margin-left: 20px;
+  margin-top: 10px;
+}
```
- [ ] Good job for implementing styling pages but please try to stick to the design as much as possible (e.g. font, colors, images, text, margins) using [the templates in Figma](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=0%3A1).

Abenezer-Tilahun commented on README.md:
```diff
@@ -1 +1,74 @@
+
+This project is [MIT](./LICENSE) licensed.
+
+_NOTE: we recommend using the [MIT license](https://choosealicense.com/licenses/mit/) - you can set it up quickly by [using templates available on GitHub](https://docs.github.com/en/communities/setting-up-your-project-for-healthy-contributions/adding-a-license-to-a-repository). You can also use [any other license](https://choosealicense.com/licenses/) if you wish._
```
- [ ] Good job for proffesional README.
- [ optional] README file should be descriptive and all main headers from the [README template](https://github.com/microverseinc/readme-template/blob/master/README.md) should be present in a your repo and please try to customize.

Abenezer-Tilahun reviewed (changes requested):
Hi @rehanullah748  

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!


## Highlights

✔️ No linter errors.
✔️ You used correct Github Flow.

![almost](https://user-images.githubusercontent.com/65860678/170491251-b1add8ba-b18f-4648-a893-20e510d957fd.gif)

## Required Changes ♻️
_Check the comments below the review._

- [x] README file should be descriptive and  all main headers from the [README template](https://github.com/microverseinc/readme-template/blob/master/README.md) should be present in your Repo. 
- [x] Please try to use meaningful Commit messages in your feature project.(at this moment you are not required to update the commit messages)
- [ ] Pull Request should have a descriptive title telling you what changes are inside and should have a summary describing introduced changes in more detail.

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

rehanullah748 committed:
images folder uploaded

images folder uploaded

rehanullah748 committed:
changes made in index.html

Reviewer required changes are made in index.html

rehanullah748 committed:
changes made in style.css

As per code reviewer changes are made in style.css

rehanullah748 committed:
data from readme file deleted

data from readme file deleted

rehanullah748 committed:
README file updated

This is my porffolio. It include my work and skills

rehanullah748 committed:
Images foler deletd

Due to some changes images folder deleted

rehanullah748 committed:
Images folder uploaded

As per code reviewer i have made some changes in images foIder i have upload upload it

rehanullah748 committed:
style.css file deleted

style.css file deleted in order to upload new changes made in style.css file as per code reviewer suggestions.

rehanullah748 committed:
style.css file uploaded

As per code reviewer suggestions after making changes in style.css file. new file is uploaded

rehanullah748 committed:
style.css file deleted

style.css file deleted

rehanullah748 committed:
style.css file created

As per code reviewer suggestions, I have made some changes in style.css file and new data is uploaded to style.css

mwanawabangona reviewed (approved):
Hi @rehanullah748,

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit:
Congratulations!  
![](https://camo.githubusercontent.com/d348d0cdaa16b202354d12402e048f52cd4d589b4ec0e55f07d62c2f22753376/68747470733a2f2f6d65646961322e67697068792e636f6d2f6d656469612f336f373246634a6d4c7a4964594a646d44652f67697068792e676966)

## Highlights
*   No linter errors :heavy_check_mark:
*   Repo looks professional :briefcase:
*   Solid pull request scroll
*   Good quality commits
*   Good job getting approved on your first try!! :clap: :clap: :clap:

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
